testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -292,6 +290,13 @@
   step %(I follow "#{node.full_hostname}")
 end
 
+Given(/^I am on the "([^"]*)" page of this "(.*?)"$/) do |page, target|
+  steps %(
+    Given I am on the Systems overview page of this "#{target}"
+  )
+  step %(I follow "#{page}" in the content area)
```

Comment:
Why not only one steps %( ?

Reply:
no idea, surely inspiration taken from another step doing it ;)